--- driver_sqlite3.c	2011-06-14 08:57:47.000000000 +0200
+++ driver_sqlite3.c.oden	2011-06-15 10:50:28.032408896 +0200
@@ -97,7 +97,7 @@ PHP_INI_BEGIN()
 	/* Path to the SQLite database file */
 	STD_PHP_INI_ENTRY("apm.sqlite_max_event_insert_timeout", "100",             PHP_INI_ALL,    OnUpdateLong,   timeout,         zend_apm_sqlite3_globals, apm_sqlite3_globals)
 	/* Max timeout to wait for storing the event in the DB */
-	STD_PHP_INI_ENTRY("apm.sqlite_db_path",                  "/var/php/apm/db", PHP_INI_ALL,    OnUpdateDBFile, db_path,         zend_apm_sqlite3_globals, apm_sqlite3_globals)
+	STD_PHP_INI_ENTRY("apm.sqlite_db_path",                  "/var/lib/php-apm", PHP_INI_ALL,    OnUpdateDBFile, db_path,         zend_apm_sqlite3_globals, apm_sqlite3_globals)
 PHP_INI_END()
 
 /* Insert an event in the backend */
